### PR TITLE
chore: add coveragereport in command

### PR DESCRIPTION
**Pull request by sarahgm**
# Description

see uncovered lines

# Reviewers:

@marigold-ui/developer

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,7 +53,7 @@
     "registry": "pnpm --filter @marigold/docs registry",
     "build-component-props": "pnpm --filter @marigold/docs build-component-props",
     "test": "jest --config jest.config.js",
-    "test:coverage": "jest --config jest.config.js --coverage --coverageReporters html",
+    "test:coverage": "jest --config jest.config.js --coverage --coverageReporters text",
```
Comment (aromko) [inline package.json:56]: Why this change is necessary?

Comment (sarahgm) [reply package.json:56]: because otherwise you can't see it? Or were is it? :D

Comment (sarahgm) [reply package.json:56]: I just did this to have the report back, see uncovered lines, etc

Comment (aromko) [reply package.json:56]: But now you just see it in the console where lines are uncovered but don't see the details right?

And with the flag `html` a directory is generated called coverage. The index.html you can open in the browser and have an interactive website with filter/search possibilities etc. Do you know that? So for me personally the html version is more handy. What do you think?

Comment (sarahgm) [reply package.json:56]: I didn't know that xD
